packages/enketo-express/public/js/src/module/translator.js

```diff
@@ -87,34 +87,44 @@ const localize = (container, lng) => {
         })
         .then(() => {
             list.forEach((el) => {
-                const key = el.dataset.i18n;
+                let key = el.dataset.i18n;
+                // translation key for element with i18nNumber is i18n-i18nNumber
+                if (el.dataset.i18nNumber) {
+                    key = `${key}-${el.dataset.i18nNumber}`;
+                }
                 if (key) {
+                    let options = {};
```

Comment:
Why did the option setting move out of the `!cache[key]` block? They only need to be set if there's going to be a new cache entry, right?

Reply:
Yes, you are right.

Fixed.

Thanks for the suggestion!